### PR TITLE
provider: Support default tags (resources aws_e* 3)

### PR DESCRIPTION
**Pull request by YakDriver**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

Reference: #7926

Output from acceptance testing: Will be handled via CI, review, and daily acceptance testing

### CODE REVIEW

File: aws/resource_aws_egress_only_internet_gateway.go [hunk 1]
```diff
@@ -22,23 +22,28 @@ func resourceAwsEgressOnlyInternetGateway() *schema.Resource {
...
 
 func resourceAwsEgressOnlyInternetGatewayCreate(d *schema.ResourceData, meta interface{}) error {
 	conn := meta.(*AWSClient).ec2conn
+	defaultTagsConfig := meta.(*AWSClient).DefaultTagsConfig
+	tags := defaultTagsConfig.MergeTags(keyvaluetags.New(d.Get("tags").(map[string]interface{})))
 
 	resp, err := conn.CreateEgressOnlyInternetGateway(&ec2.CreateEgressOnlyInternetGatewayInput{
 		VpcId:             aws.String(d.Get("vpc_id").(string)),
-		TagSpecifications: ec2TagSpecificationsFromMap(d.Get("tags").(map[string]interface{}), "egress-only-internet-gateway"),
+		TagSpecifications: ec2TagSpecificationsFromKeyValueTags(tags, ec2.ResourceTypeEgressOnlyInternetGateway),
```
Comment (YakDriver) [inline aws/resource_aws_egress_only_internet_gateway.go:46]: For ec2, this does the ignore work?

File: aws/resource_aws_eip.go [hunk 2]
```diff
@@ -146,7 +151,7 @@ func resourceAwsEipCreate(d *schema.ResourceData, meta interface{}) error {
 	}
 
 	if domainOpt == ec2.DomainTypeVpc {
-		allocOpts.TagSpecifications = ec2TagSpecificationsFromMap(d.Get("tags").(map[string]interface{}), ec2.ResourceTypeElasticIp)
+		allocOpts.TagSpecifications = ec2TagSpecificationsFromKeyValueTags(tags, ec2.ResourceTypeElasticIp)
```
Comment (YakDriver) [inline aws/resource_aws_eip.go:154]: No ignore is required?

File: website/docs/r/ecs_capacity_provider.html.markdown [hunk 3]
```diff
@@ -73,6 +73,7 @@ The `managed_scaling` block supports the following:
 In addition to all arguments above, the following attributes are exported:
 
 * `id` - The Amazon Resource Name (ARN) that identifies the capacity provider.
+* `tags_all` - A map of tags assigned to the resource, including those inherited from the provider [`default_tags` configuration block](https://www.terraform.io/docs/providers/aws/index.html#default_tags-configuration-block).
```
Comment (anGie44) [inline website/docs/r/ecs_capacity_provider.html.markdown:76]: Nit: can be moved down 1 to maintain order

File: website/docs/r/ecs_cluster.html.markdown [hunk 4]
```diff
@@ -48,6 +48,7 @@ The `default_capacity_provider_strategy` configuration block supports the follow
 In addition to all arguments above, the following attributes are exported:
 
 * `id` - The Amazon Resource Name (ARN) that identifies the cluster
+* `tags_all` - A map of tags assigned to the resource, including those inherited from the provider [`default_tags` configuration block](https://www.terraform.io/docs/providers/aws/index.html#default_tags-configuration-block).
```
Comment (anGie44) [inline website/docs/r/ecs_cluster.html.markdown:51]: similar ordering nit as above

File: website/docs/r/ecs_task_definition.html.markdown [hunk 5]
```diff
@@ -252,6 +252,7 @@ TASK_DEFINITION
 In addition to all arguments above, the following attributes are exported:
 
 * `arn` - Full ARN of the Task Definition (including both `family` and `revision`).
+* `tags_all` - A map of tags assigned to the resource, including those inherited from the provider [`default_tags` configuration block](https://www.terraform.io/docs/providers/aws/index.html#default_tags-configuration-block).
```
Comment (anGie44) [inline website/docs/r/ecs_task_definition.html.markdown:255]: similar nit as above

File: website/docs/r/efs_access_point.html.markdown [hunk 6]
```diff
@@ -60,6 +60,7 @@ The `creation_info` block supports the following:
 In addition to all arguments above, the following attributes are exported:
 
 * `id` - The ID of the access point.
+* `tags_all` - A map of tags assigned to the resource, including those inherited from the provider [`default_tags` configuration block](https://www.terraform.io/docs/providers/aws/index.html#default_tags-configuration-block).
```
Comment (anGie44) [inline website/docs/r/efs_access_point.html.markdown:63]: similar nit as above

File: website/docs/r/eip.html.markdown [hunk 7]
```diff
@@ -119,6 +119,7 @@ In addition to all arguments above, the following attributes are exported:
 
 * `id` - Contains the EIP allocation ID.
 * `private_ip` - Contains the private IP address (if in VPC).
+* `tags_all` - A map of tags assigned to the resource, including those inherited from the provider [`default_tags` configuration block](https://www.terraform.io/docs/providers/aws/index.html#default_tags-configuration-block).
```
Comment (anGie44) [inline website/docs/r/eip.html.markdown:122]: similar nit as above

**Review by anGie44 — APPROVED** [review]
Just a couple of nits in the docs changes + rebase needed, otherwise LGTM

**Pull request merged**